lib/usd/translators/shading/usdStandardSurfaceWriter.cpp

```diff
@@ -184,7 +184,7 @@ void PxrUsdTranslators_StandardSurfaceWriter::Write(const UsdTimeCode& usdTime)
         shaderSchema,
         PxrMayaUsdPreviewSurfaceTokens->NormalAttrName,
         usdTime,
-        /* ignoreIfUnauthored = */ false,
+        /* ignoreIfUnauthored = */ true,
```

Comment:
Found while debugging. Do not author an empty normals input if we are not connecting anything to it.